Formula/u/udptunnel.rb

```diff
@@ -24,9 +24,15 @@ class Udptunnel < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "8011c8ad2ef6699b3b100f259cd8e6db4ae8a799721635b06ae2a259c084c9b5"
   end
 
+  deprecate! date: "2024-03-31", because: :unmaintained
```

Comment:
Could also just be mature rather than unmaintained/abandoned.

Reply:
Not sure. There was a single release 1.0. License/doc update in 1.1. And now is essentially archived since not showing on main website and author has graduated.

Reply:
Can just update homepage for now and reconsider deprecation if homepage ever ends up entirely dead.